data/ships.txt

```diff
@@ -1164,6 +1267,49 @@ ship "Dreadnought"
 	description "The Dreadnought is a new warship that was designed by Southbound Shipyards in order to provide a large capital ship for the Free Worlds fleets, something capable of standing up to a Navy Cruiser or even a Carrier."
 
 
+ship "Dropship"
+	sprite "ship/dropship"
+	thumbnail "thumbnail/dropship"
+	attributes
+		category "Fighter"
+		"cost" 129000
+		"shields" 600
+		"hull" 200
+		"required crew" 1
+		"bunks" 9
+		"mass" 30
+		"drag" .92
+		"heat dissipation" .8
+		"cargo space" 10
+		"outfit space" 100
+		"weapon capacity" 16
+		"engine capacity" 40
+		weapon
+			"blast radius" 12
+			"shield damage" 120
+			"hull damage" 60
+			"hit force" 180
+	outfits
+		"Beam Laser" 2
```

Comment:
Wouldn't one turret (blaster, laser, or anti-missile) be more suitable than two laser guns?

Reply:
that's a valid concern, but also consider their use-case:  a shuttle is for civilian use while a dropship has to expect hostile conditions, so the comparison isn't quite fair.

Reply:
The Dropship's purpose is to drop troops, not to engage in dog-fights, therefore I don't fully understand why it has to have two fixed forward pointing cannons, the same as the Lance fighter. A small defensive turret would make more sense, or no weapons at all; cf. Boxwing.
Speaking of which, the civilian Boxwing has 800 hull, therefore I think the military Dropship deserves more than merely 200.

Reply:
I'd put some long range missiles on it.

Reply:
that would lead to it having less space to move troops, and lowering its overall speed, in its current form it can get in, drop people off while providing support if necessary, and get out, the box wing is a cargo crate with engines, and is sluggish and slow, while also being cramped and not fit for rapid movement.

Reply:
> that would lead to it having less space 

How? Two beam laser occupy 16 space, one anti-missile turret 16, a blaster turret 15. And no weapons at all would mean more room for passengers, cargo, and hull armour.

Reply:
i was talking of the thicker hull, weaponry is interchangeable in this situation, but the thicker hull would slow it down and take up space

Reply:
Boxwing + 4 outfit expansions + 3 bunk rooms: 400 shields, 800 hull, 1 crew, 13 bunks.
Proposed Dropship: 600 shields, 200 hull, 1 crew, 9 bunks.
To me the sluggish Boxwing looks both safer and more cost effective.

Reply:
I'm gonna have to side with Nescio0 on this one. Looking at the boxwing and dropship, such as it is, side by side... even when the boxwing is kitted as suggested with bunks, the speed differential is not tremendous. Though the dropship has more engine space, it doesn't do much with it out of the box, so I think it should rock chipmunks stock (losing the lasers does the trick. I also think it should indeed get an HP bump, it could just be shields, to like 200h/800s for example (same as the barb). Or more. I mean this thing is relatively a sitting duck anyway.

Also, a turret would probably prove much more useful on such a ship due to its propensity to be attacked from multiple sides while moving with purpose. IE, it's going to pickup or dropoff, it doesn't have time to be pointing fixed guns at anything. Note @Pointedstick that the V-22 has a *rear mounted* gun. Though, I think it would be fine without guns. It's not going to be doing much without support of some kind.

Reply:
For the record, stock, the boxwing and dropship have almost exactly the same acceleration and turning and the boxwing actually has about 15% more top speed